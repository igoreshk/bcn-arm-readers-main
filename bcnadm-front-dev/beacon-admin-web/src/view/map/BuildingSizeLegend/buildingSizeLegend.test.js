import React from 'react';
import { shallow } from 'enzyme';
import { BuildingSizeLegend } from './index';

const props = {
  width: 200,
  height: 150,
  translate: jest.fn()
};

const wrapper = shallow(<BuildingSizeLegend {...props} />);

describe('BuildingSizeLegend', () => {
  it('should match snapshot', () => expect(wrapper).toMatchSnapshot());
});
