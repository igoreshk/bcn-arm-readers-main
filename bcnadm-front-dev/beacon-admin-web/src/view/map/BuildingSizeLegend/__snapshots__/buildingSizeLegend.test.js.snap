// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BuildingSizeLegend should match snapshot 1`] = `
<span
  className="buidingSize"
/>
`;
